src/main/java/picard/analysis/MergeableMetricBase.java

```diff
@@ -45,6 +46,18 @@
  * merge-by-adding is only enabled for the following types: int, Integer, float, Float, double, Double, short, Short, long, Long, byte, Byte.
  * Overflow will be detected (for the short, and byte types) and an exception thrown.
  *
+ * Every (public, non-static) field in this class _must_ be @Annotated by one of: MergeByAdding, MergeByAssertEquals, NoMergingIsDerived, MergingIsManual, NoMergingKeepsValue.
+ * Static fields may be annotated, but not with @MergeByAdding.
+ *
+ * MergeByAdding: When merging another metric into this one, the value of the field in the other class will be added to the value in this.
```

Comment:
This is better although javadoc will ignore all your formatting here and present all the text as a single paragraph. If you want to fix this, I would use a description list. It uses three tags, `<dl>`, `<dt>` and `<dd>`. E.g.
```
<dl>
<dt>MergeByAdding
<dd>When merging another metric into this one, the value of the field in the other class will be added to the value in this. This will happen automatically!
<dt>MergeByAssertEquals
<dd> When merging another metric into this one, the code will assert that value of the field in the other class is equal to the value in this.
...
</dl>
```

Reply:
If you want to get even fancier you can use `@link` here to link from your dt's to the object, e.g.
```
<dt>{@link MergeByAdding MergeByAdding}
```